frontend/app/Components/BlogIndexPage/BlogCard.js

```diff
@@ -2,8 +2,11 @@ import React from 'react';
 
 const BlogCard = () => {
   return (
-    <div className="w-[20vw] border border-dashed border-2 border-gray-800 p-4 bg-gray-100 rounded-br-lg">
-      <h2 className="text-xl font-semibold mb-2 text-black">
+    <div
+      className="w-[20vw] border border-dashed border-2 border-gray-800 p-4 bg-gray-100 rounded-br-lg"
+      aria-labelledby="blog-heading"
+    >
+      <h2 className="text-xl font-semibold mb-2 text-black" id="blog-heading">
```

Comment:
Note we’re within a component that’s meant to be used multiple times on the page here, so a static id won’t be appropriate. We’ll have to update that once we switch to real data.